vendor/vendor.json

```diff
@@ -1361,6 +1361,12 @@
 			"version": "v1.2.0",
 			"versionExact": "v1.2.0"
 		},
+		{
+			"checksumSHA1": "wnEANt4k5X/KGwoFyfSSnpxULm4=",
```

Comment:
I see this package gets added but I couldn't find where it's used?

Reply:
Ah, I did delete the test that used it, but I'm going to use it again soon. The existing tests should use this to `t.FailNow()` when a test server returns `err` or the like. Apologies. So, a common idiom would be:

```go
server, err := httptest.Server()
require.Nil(err)
```

Reply:
We use assert where requires should be used much too often. +1 for adding it. But maybe in an extra PR, so we can use `requires` in the future.

Reply:
Hmmm, I just pushed new commits that make heavier use of `requires`. Do we still want to break that up into a separate PR? (I'm fine with that, not a big deal)

Reply:
I'm good with keeping it in this PR as it's now actually used.